.github/workflows/pull-request.yml

```diff
@@ -51,6 +51,8 @@ jobs:
           EP_URL: ${{ secrets.EP_URL }}
           REFRESH_TOKEN: ${{ secrets.REFRESH_TOKEN }}
           REFRESH_URL: ${{ secrets.REFRESH_URL }}
+          SF_USERNAME: ${{ secrets.SF_USERNAME }}
```

Comment:
Do we have these secrets predefined in the repo?

Reply:
Yes. Otherwise the tests will fail